models/activities/repo_activity.go

```diff
@@ -317,6 +326,23 @@ func (stats *ActivityStats) FillUnresolvedIssues(ctx context.Context, repoID int
 	return sess.Find(&stats.UnresolvedIssues)
 }
 
+func newlyCreatedIssues(ctx context.Context, repoID int64, fromTime time.Time) *xorm.Session {
+	sess := db.GetEngine(ctx).Where("issue.repo_id = ?", repoID).
+		And("issue.is_pull = ?", false).                // Retain the is_pull check to exclude pull requests
+		And("issue.created_unix >= ?", fromTime.Unix()) // Include all issues created after fromTime
+
+	return sess
+}
+
+func activeIssues(ctx context.Context, repoID int64, fromTime time.Time) *xorm.Session {
+	sess := db.GetEngine(ctx).Where("issue.repo_id = ?", repoID).
+		And("issue.is_pull = ?", false).
```

Comment:
I am a bit confused, in the issue it explicitly states:
> For PRs, it is basically the same issue.

but they are excluded right now?
So do they have the same problem or not?

Apart from that, LGTM.

Reply:
You're right, pull requests suffer from the same problem. So this fix is only for part of the problem affecting the issue counts.

However, that is unrelated to this code snippet which is counting issues, and hence should filter out pull requets.

 If this PR is approved I will be happy to submit a similar fix for pull requests.